### PR TITLE
refactor(hermes): re-organize in monorepo

### PR DESCRIPTION
**Pull request by Reisen**
This organizes hermes under a new `apps/` directory to kick-start the re-organization of the monorepo. This particular change updates all local references to the `hermes/` directory, but an additional PR will be needed to update [the documentation.](https://github.com/pyth-network/documentation/blob/main/pages/price-feeds/pythnet-price-feeds/hermes.mdx?plain=1#L10)

### CODE REVIEW

**Comment by Reisen** [comment]
Preparing PR description...

**Comment by Reisen** [comment]
Preparing PR description...

**Comment by Reisen** [comment]
Preparing PR description...

File: apps/hermes/README.md [hunk 1]
```diff
@@ -35,14 +35,14 @@ To set up and run a Hermes node, follow the steps below:
...
-   cd hermes
+   cd apps/hermes
    cargo build --release
    ```
    This will create a binary in the target/release directory.
 5. **Run the node**: To run Hermes for Pythnet, use the following command:
 
    ```bash
-   ./target/release/hermes run \
+   cargo build --release run -- \
```
Comment (ali-bahjati) [inline apps/hermes/README.md:45]: shouldn't this be `cargo run --release --` :?

**Review by ali-bahjati — APPROVED** [review]

**Review by ali-bahjati — APPROVED** [review]

**Pull request merged**